approval_frame/templates/registration/subscription_preferences.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'base.html' %}
+
+{% block head %}
+  {% load staticfiles %}
+  <script src='{% static 'approval_polls/registration_form.js' %}'></script>
+{% endblock %}
+
+{% block content %}
+
+<div class='row-fluid top-buffer'>
+  <div class='col-md-6 col-md-offset-3'>
+    <form action='/accounts/subscription/change/' method='post'>
+      {% csrf_token %}
+      <legend>Your Subscription Preferences</legend>
+      <div class='form-group {% if form.errors.zipcode %} has-error {% endif %}'>
+        {% for error in form.errors.zipcode %}
+          <label class='control-label' for='{{ form.zipcode.id_for_label }}'>
+              {{ error }}
+          </label>
+        {% endfor %}
+        <input type='text' class='form-control' id='{{ form.zipcode.id_for_label }}'
+               name='{{ form.zipcode.name }}' placeholder='{{ form.zipcode.label }}'
+               value='{{ form.zipcode.value|default_if_none:'' }}' maxlength={{ form.zipcode.field.max_length }}>
+      </div>
+      <div class='form-group {% if form.errors.newslettercheckbox %} has-error {% endif %}'>
+        {% for error in form.errors.newslettercheckbox %}
+          <label class='control-label' for='{{ form.newslettercheckbox.id_for_label }}'>
+            {{ error }}
+          </label>
+        {% endfor %}
+        <div class='checkbox'>
+          <label>
+            <input type='checkbox' id='{{ form.newslettercheckbox.id_for_label }}'
+              name='{{ form.newslettercheckbox.name }}' {% if boxchecked %} checked {% endif %}>
+              Subscribe to our Newsletter
+          </label>
+        </div>
+      </div>
+      <button class='btn btn-success' type='submit'>Update Your Subscription</button>
```

Comment:
@balajisubr, Something minor, but I think it would be better if you could keep it consistent with 'Update my Username' etc.. to change it to 'Update my Subscription Preference'